File: DSA/NiharikaChand_590017697/Day1Question2.java
import java.util.*;
class Day1Question2
{
    public static void main(String args[]) 
    {
        Scanner sc = new Scanner(System.in);
        int arr [] = {5,3,6,2,2,4};
        int n = arr.length;
        int i;
        int freq[] = new int[n];
        for (i = 0; i < n; i++) 
        {
            freq[arr[i]-1]++;
        }

        // Find missing and repeated numbers
        int m= -1, r = -1;
        for (i = 0; i < n; i++) 
        {
            if (freq[i] == 0) 
            {
                m = i+1;
            } 
            else if (freq[i] > 1) 
            {
                r = i+1; 
            }
        }

        System.out.println("Missing element: " + m);
        System.out.println("Repeated element: " + r);
    }
}

File: DSA/Stuti_Jain_590012149/Day_1_Question_2.c
#include<stdio.h>
int i, n;
int main() {
    printf("Enter the size of the array: \n");
    scanf("%d", &n);

    int arr[n];
    int freq[n+1];

    for(i=0; i<n; i++){
        freq[i]=0;
    }

    printf("Enter the elements of the array: \n");
    for(i=0; i<n; i++){
        scanf("%d", &arr[i]);
        freq[arr[i]]++;

    }

    int missing=0;
    int repeated=0;

    for(i=1; i<=n; i++){
        if(freq[i]==0){
        missing=i;
    }
        else if(freq[i]==2){
        repeated=i;
    }
    }

    printf("The missing elements is: %d\n", missing);
    printf("The repeated element is: %d\n", repeated);


}
